contrib/builder/deb/generate.sh

```diff
@@ -58,6 +58,8 @@ for version in "${versions[@]}"; do
 		libdevmapper-dev # for "libdevmapper.h"
 		libltdl-dev # for pkcs11 "ltdl.h"
 		libsqlite3-dev # for "sqlite3.h"
+		libseccomp2  # for "libseccomp.so.2"
```

Comment:
Doesn't `libseccomp-dev` pull this in properly already?  Did we need it to be explicit for some other reason?